pallets/dia-oracle/src/weights.rs

```diff
@@ -48,106 +46,212 @@ pub trait WeightInfo{
 	fn authorize_account() -> Weight {
-		Weight::from_ref_time(1_241_248_000)
+		// Proof Size summary in bytes:
+		//  Measured:  `150`
+		//  Estimated: `3615`
+		// Minimum execution time: 12_000_000 picoseconds.
+		Weight::from_parts(13_000_000, 0)
+			.saturating_add(Weight::from_parts(0, 3615))
 			.saturating_add(T::DbWeight::get().reads(1))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
-	// Storage: DiaOracle AuthorizedAccounts (r:2 w:1)
+	/// Storage: DiaOracleModule AuthorizedAccounts (r:2 w:1)
+	/// Proof Skipped: DiaOracleModule AuthorizedAccounts (max_values: None, max_size: None, mode: Measured)
 	fn authorize_account_signed() -> Weight {
-		Weight::from_ref_time(1_525_600_000)
+		// Proof Size summary in bytes:
+		//  Measured:  `187`
+		//  Estimated: `6127`
+		// Minimum execution time: 14_000_000 picoseconds.
+		Weight::from_parts(15_000_000, 0)
+			.saturating_add(Weight::from_parts(0, 6127))
 			.saturating_add(T::DbWeight::get().reads(2))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
-	// Storage: DiaOracle AuthorizedAccounts (r:1 w:0)
+	/// Storage: DiaOracleModule AuthorizedAccounts (r:1 w:0)
+	/// Proof Skipped: DiaOracleModule AuthorizedAccounts (max_values: None, max_size: None, mode: Measured)
 	fn deauthorize_account() -> Weight {
-		Weight::from_ref_time(276_664_000)
+		// Proof Size summary in bytes:
+		//  Measured:  `150`
+		//  Estimated: `3615`
+		// Minimum execution time: 6_000_000 picoseconds.
+		Weight::from_parts(7_000_000, 0)
+			.saturating_add(Weight::from_parts(0, 3615))
 			.saturating_add(T::DbWeight::get().reads(1))
 	}
-	// Storage: DiaOracle AuthorizedAccounts (r:2 w:1)
+	/// Storage: DiaOracleModule AuthorizedAccounts (r:2 w:1)
+	/// Proof Skipped: DiaOracleModule AuthorizedAccounts (max_values: None, max_size: None, mode: Measured)
 	fn deauthorize_account_signed() -> Weight {
-		Weight::from_ref_time(1_513_398_000)
+		// Proof Size summary in bytes:
+		//  Measured:  `187`
+		//  Estimated: `6127`
+		// Minimum execution time: 14_000_000 picoseconds.
+		Weight::from_parts(15_000_000, 0)
+			.saturating_add(Weight::from_parts(0, 6127))
 			.saturating_add(T::DbWeight::get().reads(2))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
-	// Storage: DiaOracle AuthorizedAccounts (r:1 w:0)
-	// Storage: DiaOracle CoinInfosMap (r:0 w:1)
+	/// Storage: DiaOracleModule AuthorizedAccounts (r:1 w:0)
+	/// Proof Skipped: DiaOracleModule AuthorizedAccounts (max_values: None, max_size: None, mode: Measured)
+	/// Storage: DiaOracleModule CoinInfosMap (r:0 w:1)
+	/// Proof Skipped: DiaOracleModule CoinInfosMap (max_values: None, max_size: None, mode: Measured)
 	fn set_updated_coin_infos() -> Weight {
-		Weight::from_ref_time(1_241_248_000)
+		// Proof Size summary in bytes:
+		//  Measured:  `187`
+		//  Estimated: `3839`
+		// Minimum execution time: 15_699_000_000 picoseconds.
+		Weight::from_parts(15_772_000_000, 0)
```

Comment:
This weight is probably problematically high

Reply:
reverted the weights. [514ad70](https://github.com/pendulum-chain/oracle-pallet/pull/9/commits/514ad70472e4b3fb9ad83751c23f9af9710cf9f7)